### DISCUSSION TITLE
Adapter Concurrency of

### DISCUSSION THREAD

**Discussion by luckygreen** [Koenkk/zigbee2mqtt#12187] [Devices]
What is the correct Adapter Concurrency setting for a CC2652P2?

I understand from reading the[ Zigbee2MQTT docs](https://www.zigbee2mqtt.io/guide/configuration/adapter-settings.html#transmitter-power) and reading #10664 what adapter_concurreny does.

However, even after reading through the [CC2652P2 data sheet ](https://www.ti.com/document-viewer/CC2652P/datasheet)twice, I still have not figured out what the concurrency value for this part is and which specific value I should therefore enter into [configuration.yaml.](https://www.zigbee2mqtt.io/guide/configuration/)

Do you happen to know the answer?

Thanks!
-- Lucky

**Comment by Koenkk** [comment]
The default one should be good. It is not a setting in the CC2652P firmware, it just limits how many concurrent zigbee commands are send; where 1 zigbee command is the command itself until the reply (confirmation) is received
